cluster/swarm/node.go

```diff
@@ -265,7 +265,7 @@ func (n *node) updateContainer(c dockerclient.Container, containers map[string]*
 		}
 		container.Info = *info
 		// real CpuShares -> nb of CPUs
-		container.Info.Config.CpuShares = container.Info.Config.CpuShares / 100.0 * n.Cpus
+		container.Info.Config.CpuShares = container.Info.Config.CpuShares / 1024.0 * n.Cpus
```

Comment:
@smothiki it should be `container.Info.Config.CpuShares = container.Info.Config.CpuShares * 1024.0 / n.Cpus`